kayenta-newrelic-insights/src/main/java/com/netflix/kayenta/newrelic/canary/NewRelicCanaryScope.java

```diff
@@ -26,4 +27,8 @@
 @ToString(callSuper = true)
 public class NewRelicCanaryScope extends CanaryScope {
 
+  @NotNull
+  private String scopeKey;
+
+  private String appName;
```

Comment:
Is this required for all New Relic integrations with Insights or specific to your use case?